sdk/keyvault/tests.yml

```diff
@@ -11,7 +11,7 @@ stages:
           SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
         Canary:
           SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-          Location: 'eastus2euap'
+          Location: 'eastus2'
```

Comment:
Why it's necessary to change location?

Reply:
When testing in Canary Cloud, a deploy resource exception occurred. According to the description of the error message, it was judged that the location setting was incorrect.

Reply:
Okay, the `Canary` test entry has been completely deleted in new version.